lib/push-jenkins-update.js

```diff
@@ -77,20 +77,16 @@ function findPrInRef (gitRef) {
   return parseInt(gitRef.split('/')[2], 10)
 }
 
-async function findLatestCommitInPr (options, pageNumber = 1) {
-  const res = await githubClient.pulls.listCommits({
+async function findLatestCommitInPr (options) {
+  const { Octokit } = require('@octokit/rest')
+  const githubClient = new Octokit({})
```

Comment:
Was it deliberate to not use `githubClient` from line 3?

Reply:
That was probably a blunder. Sorry about that!

Reply:
Is this why we're hitting rate limits? The github-bot logs now have lots of "API rate limit exceeded for 23.253.100.79. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)" errors, e.g.
```console
{"name":"bot","hostname":"infra-rackspace-debian8-x64-1","pid":15899,"req_id":"e62c49d0-bbaa-11ed-9410-77b81a7c3334","identifier":"node-test-linux-ubuntu1804-64","event":"end","level":50,"err":{"message":"API rate limit exceeded for 23.253.100.79. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","name":"HttpError","stack":"HttpError: API rate limit exceeded for 23.253.100.79. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\n    at /home/iojs/github-bot/node_modules/@octokit/request/dist-node/index.js:86:21\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)\n    at async Object.next (/home/iojs/github-bot/node_modules/@octokit/plugin-paginate-rest/dist-node/index.js:112:28)\n    at async findLatestCommitInPr (/home/iojs/github-bot/lib/push-jenkins-update.js:88:23)","code":403},"msg":"Error while handling Jenkins end event","time":"2023-03-05T23:10:20.025Z","v":0}
```

Reply:
Ugh, sending a PR momentarily. I wouldn't have put these things together. Sorry about that.

Reply:
Thanks for opportunity to learn why Github requests better be authenticated 👍🏻